packages/block-library/src/button/style.scss

```diff
@@ -47,7 +47,7 @@ $blocks-button__height: 56px;
 	border-radius: 0 !important;
 }
 
-.wp-block-button.is-style-outline .wp-block-button__link,
+.is-style-outline .wp-block-button__link,
```

Comment:
This is very weird to me because potentially a parent group block might use this modifier classname for something else, triggering this rule inadvertently? Are there other alternatives?

Reply:
Right. I thought reverting to the previous state was the safest path. However, I like the sound of what you say and would like to be more specific as well.

Reply:
I've created #24919 as a follow-up to implement the change I suggested in my previous comment.